templates/services/ntopng/ntopng_yml.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+mkdir -p /srv/ntopng
+
+# create ntopng.yml
+{
+  echo "services:"
+  echo "  ntopng:"
+  echo "    image: jonbackhaus/ntopng"
+  echo "    ports:"
+  echo "      - \"8090:8090\""
+  echo "    volumes:"
+  echo "      - \"/var/run/docker.sock:/var/run/docker.sock\""
+  echo "      - \"/ntopng_data:/data jonbackhaus/ntopng\""
+  echo "version: \"2\""
+  echo "volumes:"
+  echo "  ntopng_data:"
```

Comment:
@sravya71 there is a better way to do this ...

also `/data jonbackhaus/ntopng` is not a directory